jupyterhub/jupyterhub/base/jupyterhub-dc.yaml

```diff
@@ -114,18 +139,23 @@ spec:
               key: POSTGRESQL_PASSWORD
         - name: JUPYTERHUB_DATABASE_HOST
           value: jupyterhub-db
-        image: quay.io/odh-jupyterhub/jupyterhub-img:v0.3.1
+        image: quay.io/odh-jupyterhub/jupyterhub-img:pr-104
         imagePullPolicy: "Always"
         name: wait-for-database
         resources:
           limits:
+            cpu: "1"
+            memory: 1Gi
+          requests:
```

Comment:
The init container definitiely does not need 1 cpu:-) But it does not really matter since it's gone quickly, but I'd still say change it to make things reasonable and consistent